spikeinterface/widgets/activity.py

```diff
@@ -31,6 +31,8 @@ class PeakActivityMapWidget(BaseWidget):
         Plot rates with contact colors
     with_interpolated_map: bool (default True)
         Plot rates with interpolated map
+    with_channel_ids: bool False default
+        add channel ids text on the probe
```

Comment:
```suggestion
        Add channel ids text on the probe
```
Can you make them all capital in the docstrings?

Reply:
Of course camarade. lets make it clean